docs/installation/server-config/security.rst

```diff
@@ -132,28 +133,130 @@ Configuring Authorization of KSQL with Kafka ACLs
 Kafka clusters can use ACLs to control access to resources. Such clusters require each client to authenticate as a particular user.
 To work with such clusters, KSQL must be configured to :ref:`authenticate with the Kafka cluster <config-security-ssl-sasl>`,
 and certain ACLs must be defined in the Kafka cluster to allow the user KSQL is authenticating as access to resources.
-The list of ACLs that must be defined depends on whether the KSQL cluster is configured for
-:ref:`interactive <config-security-ksql-acl-interactive>` or :ref:`non-interactive (headless) <config-security-ksql-acl-headless>`.
+The list of ACLs that must be defined depends on the version of the Kafka cluster.
+
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Kafka Cluster v2.0 and above
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Kafka 2.0 saw some enhancements that drastic simplifies the ACLs required to run KSQL against a Kafka cluster secured with ACLs.
```

Comment:
Grammar: drastic -> drastically

But I'd rather suggest to reword to (incl. hyperlinks):

> Kafka 2.0 simplifies the ACLs required to run KSQL against a Kafka cluster secured with ACLs (see [KIP-277](https://cwiki.apache.org/confluence/display/KAFKA/KIP-277+-+Fine+Grained+ACL+for+CreateTopics+API) and [KIP-290](https://cwiki.apache.org/confluence/display/KAFKA/KIP-290%3A+Support+for+Prefixed+ACLs) for details). It is highly recommended to use Kafka versions of 2.0 or above for deploying secure installations of Kafka and KSQL.

The original text reads more like an op-ed ;-)